Gemfile

```diff
@@ -25,7 +25,7 @@ group :development do
   gem "simplecov", "~> 0.22"
   gem "simplecov-console", "~> 0.9", ">= 0.9.3"
   gem "standard", "~> 1.45.0"
-  gem "steep", "~> 1.9.4"
+  gem "steep", "~> 1.10.0.pre", require: false
```

Comment:
We don't use `Bundler.require` anywhere so I don't think `require: false` makes any difference--none of the gems are auto-required by bundler.

Reply:
OK. In fact the `require: false` is to skip loading RBS files from Steep and its dependency. But I think we can do almost the same by adding `ignore: true` entries in `rbs_collection.yaml`.

Reply:
> In fact the `require: false` is to skip loading RBS files from Steep and its dependency.

TIL. Do steep or RBS read the `Gemfile` and the `require:` value? I only knew of what bundler does with `require:` which I thought only had an effect when you call `Bundler.require`.

Reply:
https://github.com/ruby/rbs/wiki/Release-Note-3.0#rbs-collection-skips-loading-application-gems

Yeah, this is a kind of abuse of internal API... RBS reads the `Bundler.definition`, loaded from `Gemfile`.
I'd keep the `rbs_collection.yaml` config.